===================================================================
RCS file: /var/lib/cvsd/home/cvsroot/telecom/yate/modules/zapchan.cpp,v
retrieving revision 1.26
retrieving revision 1.27
diff -u -p -r1.26 -r1.27
--- yate/modules/zapchan.cpp	2005/01/16 04:39:36	1.26
+++ yate/modules/zapchan.cpp	2005/01/24 08:37:37	1.27
@@ -590,7 +590,7 @@ PriSpan::PriSpan(struct pri *_pri, int s
       m_restart(0), m_chans(0), m_ok(false)
 {
     Debug(DebugAll,"PriSpan::PriSpan(%p,%d,%d,%d) [%p]",_pri,span,chans,fd,this);
-    ZapChan **ch = new (ZapChan *)[chans];
+    ZapChan **ch = new ZapChan* [chans];
     for (int i = 1; i <= chans; i++)
 	ch[i-1] = (i == dchan) ? 0 : new ZapChan(this,i,s_buflen);
     m_chans = ch;
